python/sglang/srt/models/deepseek_v2.py

```diff
@@ -1009,6 +1054,127 @@ def forward_absorb_fused_mla_rope(
 
         return output
 
+    def _chunked_prefix_attn_mha(
+        self,
+        q: torch.Tensor,
+        accum_output: torch.Tensor,
+        accum_lse: torch.Tensor,
+        forward_batch: ForwardBatch,
+    ) -> torch.Tensor:
+
+        assert is_flashinfer_available()
+        from flashinfer.cascade import merge_state
```

Comment:
Somehow I found the merge_state implementation in flashinfer has some accuracy gap with vllm v1 [merge_attn_state](https://github.com/vllm-project/vllm/blob/d085a4408244502dd496cae0ccbc524352e87ef8/tests/kernels/test_cascade_flash_attn.py#L10) do you wanna try this one and see if it gives slight better results (without cuda graph the diff is slightly smaller but still have some gap)? cc @zhyncs

Reply:
ref https://github.com/vllm-project/vllm/pull/16173/files